.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,5 +19,11 @@ Please provide the following information:
 
 **- How to verify it**
 
+**- Description for the changelog**
+
+Write a short (one line) summary that describes the changes in this
+pull request for inclusion in the changelog:
```

Comment:
I suggest enclosing this prompt in a comment, like the info above. Otherwise, I suspect this text will appear in a lot of pull requests.